codebase/superdarn/src.lib/tk/hmb.1.0/src/hmb.c

```diff
@@ -183,7 +183,7 @@ void make_hmb()
 
       mlt=m;
 
-      lathmb[n]=n+50.0;
+      lathmb[n]=n+40.0;
```

Comment:
```suggestion
      lathmb[n]=n+40.0; /* The +40.0 means we start making the grid used by the iterative HMB-finding algorithm at geomagnetic latitude of 40 degrees. /*  
```

Reply:
And this is also in response to @aburrell 's suggestion for a comment here.

Reply:
This change looks good to me.  Should I commit it?

Reply:
Same thing here as I understand why it is +40.0.  Feel free to add the commit and resolve the conversation.

Reply:
Wait one sec - I got the closing of the comment wrong! ;) Should be `*/` not `/*` at the end - right?

Reply:
I think I fixed it... Sorry about that typo!